.github/workflows/publish.yml

```diff
@@ -6,6 +6,9 @@ on:
         description: Dry-Run
         required: false
         type: boolean
+  push:
```

Comment:
I don't think we have a v2 branch anymore.

Reply:
Removed.